ons_alpha/topics/models.py

```diff
@@ -41,19 +44,81 @@ class TopicPage(BaseTopicPage):
     parent_page_types = ["topics.TopicSectionPage"]
     subpage_types = ["articles.ArticleSeriesPage", "bulletins.BulletinSeriesPage", "methodologies.MethodologyPage"]
     page_description = "A specific topic page. e.g. Public sector finance or Inflation and price indices"
+    summary = models.CharField(blank=True, max_length=255)
+    topic_page_nav = models.CharField(blank=True, max_length=255)
+
+    content_panels = BasePage.content_panels + [
+        FieldPanel("summary"),
+        FieldPanel("topic_page_nav"),
+    ]
+
+    edit_handler = TabbedInterface(
+        [
+            ObjectList(content_panels, heading="Content"),
+            ObjectList(
+                [FieldPanel("topic")],
+                help_text="Select the topics that this page relates to.",
+                heading="Taxonomy",
+            ),
+            ObjectList(BasePage.promote_panels, heading="Promote"),
+        ]
+    )
 
     def clean(self):
         super().clean()
         if TopicSectionPage.objects.filter(topic=self.topic).exists():
             raise ValidationError({"topic": "Topic Section Page with this topic already exists."})
 
+    def topic_list(self):
```

Comment:
I'm not sure calling the result a `subpage_dict` is the best solution either. Is it just meant to be the child topics and topic sections? Dedicated methods to get each is probably better, rather than making it too generic.